src/govuk/settings/_links.scss

```diff
@@ -4,43 +4,62 @@
 
 /// Enable new link styles
 ///
+/// If enabled, the link styles will change. Underlines will:
+///
+/// - be consistently thinner and a bit further away from the link text
+/// - have a clearer hover state, where the underline gets thicker to make the
+///   link stand out to users
+///
+/// These changes are an opt-in feature because you may:
```

Comment:
This feels a little like we're explaining the feature from our perspective rather than our users'. Can you make it more task-based?

For example, something like:

You should only enable the new link styles if both:

- you've made sure your whole service will use the new style consistently
- you do not have links in a multi-column CSS layout - there's [a Chromium bug that affects links]